src/httpbeast.nim

```diff
@@ -69,6 +69,11 @@ template handleAccept() =
   let (client, address) = fd.SocketHandle.accept()
   if client == osInvalidSocket:
     let lastError = osLastError()
+
+    if lastError.int32 == EMFILE:
+      echo osErrorMsg(lastError)
```

Comment:
Please use `warn` here from the `logging` module and expand the message a little.

```suggestion
      warn("Ignoring EMFILE error: ", osErrorMsg(lastError))
```